department-of-veterans-affairs/abd-vro
# MCP-1827: V2 asthma

# V2 Update for Asthma

## Description



### What was the problem?

More information is available and should be added to the claim to be used as evidence. 



### How does this fix it?

Proposal to rename `asthmaRelevant` to `conditionRelated` so it can be reused for all conditions.

### Classification of Prescribed Medications
`suggestedCategory` is included in the medication object to categorize medications by class. This information is required for RVSR's to enter into the calculator. We can provide an accurate mapping of Asthma related medications to their medication classes. 

This PR also adds the Condition FHIR resource made available by the Lighthouse API. Active conditions are filtered for relevance to asthma. 

note: SNOMEDCT and ICD-10 codes are included in the same sets because they will not intersect. 



### Jira Tickets



- [MCP-1827](https://amida.atlassian.net/browse/MCP-1827)

## How to test this PR

- Step 1 
`./gradlew -PenablePrototype=true build check docker`:
-`export COMPOSE_PROFILES=prototype`
- `./gradlew :app:dockerComposeDown :app:dcPrune :app:dockerComposeUp`


- Step 2 
- go to Swagger page
- enter "6602v2" for `diagnosticCode`

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
